indico/web/forms/fields.py

```diff
@@ -865,3 +868,33 @@ def render_protection_message(self):
 
 class IndicoTagListField(HiddenFieldList):
     widget = JinjaWidget('forms/tag_list_widget.html', single_kwargs=True)
+
+
+class IndicoMarkdownField(TextAreaField):
+    widget = JinjaWidget('forms/markdown_widget.html', single_kwargs=True, rows=15)
+
+
+class IndicoTimezoneSelectField(SelectField):
+    def __init__(self, *args, **kwargs):
+        super(IndicoTimezoneSelectField, self).__init__(*args, **kwargs)
+        config = Config.getInstance()
+        self.choices = [(v, v) for v in pytz.common_timezones]
+        self.default = config.getDefaultTimezone()
+
+    def process_data(self, value):
+        try:
```

Comment:
why not call `super()`?

    def process_data(self, value):
        try:
            self.data = self.coerce(value)
        except (ValueError, TypeError):
            self.data = None

(from SelectField - exactly what you have here)